docs/UserGuide.md

```diff
@@ -110,8 +114,9 @@ Deletes the specified item from the list of tracked inventory.
 Format: `deletei INDEX`
 
 * Deletes the item at the specified `INDEX`.
-* The index refers to the index number shown in the displayed inventory list.
-* The index **must be a positive integer** 1, 2, 3, …​
+
+* `INDEX` refers to the index number shown in the displayed inventory list.
+* `INDEX` **must be a positive integer** 1, 2, 3, …​
 * Deleting items with active orders is not permitted.
```

Comment:
I would suggest changing this to something like

`TrackO does not allow items that are currently involved with unpaid/undelivered orders to be deleted` to make it sound clearer